import_broadbands.py

```diff
@@ -44,7 +44,7 @@ def process_row(self, row):
 @click.option('--filename', help='Broadbands *.csv file')
 @click.option(
     '--apiurl',
-    default='http://localhost:8000/api/broadband/', help='API url')
+    default='http://localhost:8000/api/broadbands/', help='API url')
```

Comment:
Not sure broadbands is the plural of broadband.  But I guess it's okay as it is consistent.  I guess it'd be broadbandrecord or something.